server/webapp/WEB-INF/rails.new/app/presenters/api_v1/config/config_repo/materials/git_material_representer.rb

```diff
@@ -27,8 +27,6 @@ class GitMaterialRepresenter < ScmMaterialRepresenter
                    setter: lambda { |value, options|
                      value.blank? ? self.setBranch('master') : self.setBranch(value)
                    }
-          property :submodule_folder
-          property :shallow_clone
```

Comment:
That looks like breaking compatibility?

Reply:
@ketan -- firstly, these fields were never documented.
Even if these fields are specified, those were not in use for `config repo` material! So removing these fields won't lead to broken functionality.

We've added an `xsl` migration to wipe out unnecessary fields from the material tag.

Reply:
`submodule_folder` was a wrongly added attribute, `shallow_clone` needs to come back.